### DISCUSSION TITLE
Copilot website: 'Start Free Trial' not working

### DISCUSSION THREAD

**Discussion by amebru** [community/community#51988] [Copilot Conversations]
### Select Topic Area

Bug

### Body

On the [Copilot site](https://github.com/settings/copilot), 'Start Free trial' isn't working for me.

https://user-images.githubusercontent.com/2973412/229901294-d682bb60-5143-464d-a405-e04425e8df37.mov

**Comment by Sparros** [comment]
I have the same issue. Do you have the education pack or using a new machine?

**Reply by hongdavid94** [reply]
I have the same issue and have the education pack

**Reply by akadirdagaoturan** [reply]
i have the same issue and student pack.

**Comment by iamcjwilliams** [comment]
@amebru @Sparros Our copilot team is currently looking into this issue

**Reply by jcavell** [reply]
The free trial doesn't work. It says it will charge you immediately. Surely this is an urgent bug? How can I have trust in Copilot if you can't even get a free trial working?

**Comment by zedchamaa** [comment]
Did you try it in Chrome?

**Reply by hongdavid94** [reply]
yes I tried it on chrome

**Comment by nleigh** [comment]
I have the same issue

**Reply by LeoL20** [reply]
Same here with student pack,

**Comment by SRwenzi** [comment]
请问解决了吗

**Reply by CyAScott** [reply]
I have a similar issue. I have copilot via my organization. However, I also want copilot as an individual. The links to start a free trail do not work with the mobile app. The links to start the trail just take me to my copilot settings which does not give me an option to start a trail. It only shows me info about my copilot subscription via my organization.

**Comment by luck-xj** [comment]
I also have the same situation. Is there any solution to this problem? Please help me 🙏

**Comment by j3rrys** [comment]
Same issue but i just but i subscibe to a 10$/month individual account today to try the 30day free trial all step has been checked successfully  but when i log into VS and when I go on the Copilot site it shows free trial expired
